src/CipherSweet.php

```diff
@@ -11,7 +11,7 @@
 use ParagonIE\CipherSweet\CompoundIndex;
 use ParagonIE\CipherSweet\Constants;
 use ParagonIE\CipherSweet\Contract\TransformationInterface;
-use ParagonIE\CipherSweet\EncryptedRow;
+use ParagonIE\CipherSweet\EncryptedField;
```

Comment:
Why is the logic being updated to migrate away from `EncryptedRow` (which is almost certainly what you want) to the legacy `EncryptedField`?